index.js

```diff
@@ -193,6 +196,21 @@ function onchange (match, command, rawArgs, opts = {}) {
   watcher.on('error', (error) => log(`watcher error: ${error}`))
 }
 
+function getIgnoreMergedFromIgnoreFile(exclude = [], ignorePath) {
+    if(existsSync(ignorePath)) {
```

Comment:
I'm a little concerned about adding a synchronous code path to this module, it's technically a breaking change for anyone using this as a module instead of CLI. Can we make the `ignore` feature opt-in instead of default?

Reply:
Well thought. I will remove the default

Reply:
Alternatively you could put this logic in the CLI itself, that would be fine.

Reply:
Move logic to CLI